whitesource/util.py

```diff
@@ -353,11 +357,13 @@ def print_scans(
 
 
 def send_mail(
-    notification_recipients: typing.List[str],
+    notification_recipients: typing.Union[None, typing.List[str]],
     cve_threshold: float,
     product_name: str,
     projects: typing.List[whitesource.model.WhiteSrcProject],
 ):
+    if not notification_recipients:
+        logger.warning('No recipients defined. No emails will be sent...')
 
     if len(notification_recipients) > 0:
```

Comment:
That will not work for `None`. Also, I don't see the call being adjusted anywhere - can notification_recipients actually _be_ `None` (today)?